include/jemalloc/internal/sz.h

```diff
@@ -309,6 +309,17 @@ sz_size2index_usize_fastpath(size_t size, szind_t *ind, size_t *usize) {
 	}
 }
 
+JEMALLOC_ALWAYS_INLINE size_t
+sz_s2u_size_class_compute(size_t size) {
```

Comment:
nit: maybe `sz_s2u_compute_using_delta`? Since it's logically part of sz_s2u_compute, I'd keep the prefix the same.